package.json

```diff
@@ -7,7 +7,13 @@
   "unpkg": "./umd/index.all.min.js",
   "types": "index.d.ts",
   "author": "atomiks",
+  "contributors": [
+    "Brett Zamir"
+  ],
   "license": "MIT",
+  "bugs": "https://github.com/atomiks/tippyjs/issues",
+  "homepage": "https://atomiks.github.io/tippyjs/",
+  "engines": {},
```

Comment:
Leaving this empty is not very helpful, in fact npm docs suggest to set it if the engines option is present https://docs.npmjs.com/files/package.json#engines

Reply:
Hmm, ok. With these lines:

> If you specify an “engines” field, then npm will require that “node” be somewhere on that list. If “engines” is omitted, then npm will just assume that it works on node.

I guess I was confused because it doesn't end with "assume that it works on any version of Node", so I guess I had misread that as suggesting that to indicate no Node support (as in browser only), one should include it. It seems a more clear way to indicate no Node support too, but it appears one is forced to imply it works on Node. :(

Shall I revert then?

Reply:
I've asked at https://npm.community/t/engines-and-non-node-usage/6478

Reply:
Though if it takes a while to get a response or resolution, maybe I can just revert for now (or keep as is if you prefer), and possibly add it back in later in another PR if a clarification or change is forthcoming.